src/hotspot/cpu/aarch64/aarch64_neon.ad

```diff
@@ -238,6 +305,17 @@ instruct vcvt4Sto4I(vecX dst, vecD src)
   ins_pipe(pipe_class_default);
 %}
 
+instruct vcvt2Sto2I(vecD dst, vecD src)
```

Comment:
Do we really need this rule? There still have some mid-end issues remained[1] that some vector sizes could be marked as unsupported in advance.  

I suppose some rules are useless after that fix.

[1] https://github.com/openjdk/jdk/pull/3507/files#r623691031

Reply:
This rule has fixed the problem you mentioned. As for other problems, i'm not sure whether they still exist.

Reply:
Currently, we are *not* going to support small vector lane count which less than 64bits[1][2]. I prefer to fix the mid-end problem first and remove this kind of rules in this patch.
 
[1] https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/matcher.hpp#L341
[2] https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/aarch64/aarch64.ad#L2491

Reply:
Hi, The vector cast rule below 64bit has been deleted, and the patch to fix the mid-end problem has been submitted in https://github.com/openjdk/jdk/pull/5160. Thanks for your suggestion.